app/models/student.rb

```diff
@@ -6,4 +6,8 @@ class Student < ApplicationRecord
   has_many :coordinators, through: :enrollments
 
   validates :first_name, presence: true, length: { minimum: 3 }
+
+  def name
```

Comment:
You can define a module for these and then include that module in the classes if you want.